sql/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM postgres:alpine3.13
+
+ENV POSTGRES_PASSWORD=password
```

Comment:
I am not sure what proper procedure is for things such as this. Local test dbs don't feel like they need very secured?